src/main/resources/static/js/MainWindow.js

```diff
@@ -1,769 +1,708 @@
-		$
-				.notify(
-						{
-							message : "Please choose a time zone before saving the configuration."
-						}, {
-							allow_dismiss : true,
-							type : "info",
-							placement : {
-								from : "top",
-								align : "left"
-							},
-							animate : {
-								enter : "animated fadeInDown",
-								exit : "animated fadeOutUp"
-							}
-						});
-		return false;
-	} else if (date.length == 0) {
-		$
-				.notify(
-						{
-							message : "Please specify where to find the date in your file before saving the configuration."
-						}, {
-							allow_dismiss : true,
-							type : "info",
-							placement : {
-								from : "top",
-								align : "left"
-							},
-							animate : {
-								enter : "animated fadeInDown",
-								exit : "animated fadeOutUp"
-							}
-						});
-		return false;
-	} else if (streams.length == 0) {
-		$
-				.notify(
-						{
-							message : "Please add a datastream before saving the configuration."
-						}, {
-							allow_dismiss : true,
-							type : "info",
-							placement : {
-								from : "top",
-								align : "left"
-							},
-							animate : {
-								enter : "animated fadeInDown",
-								exit : "animated fadeOutUp"
-							}
-						});
-		return false;
-	}
-
-	var map = mappingData;
-
-	var cfgName = prompt("Name of Configuration:");
-	if (cfgName == null) {
-		return false;
-	}
-	if (cfgName == "") {
-		addToLog("A configuration needs a name");
-		return false;
-	}
-
-	var formData = {
-		name : cfgName,
-		delimiter : currentDelimiter,
-		numberOfHeaderlines : parseInt(currentHeaderLines, 10),
-		timezone : $("#selecttime option:selected").attr("data-value"),
-		dateTime : date,
-		streamData : streams,
-		mapOfMagicNumbers : map,
-		dataType : filetype,
-		frostURL : url
-	};
-
-	var jsoncfg = JSON.stringify(formData, null, 4);
-
-	$.ajax({
-		type : "POST",
-		url : "config/create",
-		contentType : "application/json",
-		data : jsoncfg,
-		dataType : "json",
-		error : function(e) {
-			addToLog(e.responseText);
-		},
-		success : function(e) {
-			var option = new Option(e.name, e.name, null, null);
-			option.setAttribute("data-value", JSON.stringify(e, null, 4));
-			$("#configs").append(option).val(e.name).trigger("change");
-			addToLog("Configuration saved.");
-		}
-	});
+  var currentInput;
+  var parsed;
+  var stop = false;
+  var url = document.querySelector("#frostserverurl").value;
+
+  try {
+    new RegExp(currentDelimiter);
+    if (currentDelimiter == null || currentDelimiter == "") {
+      currentDelimiter = "\n";
+    }
+  } catch (e) {
+    currentDelimiter = "\n";
+  }
+
+  if (!/^0*[0-9]{1,4}$/.test(currentHeaderLines)) {
+    currentHeaderLines = 0;
+  }
+
+  var date = [];
+  $("#timeTable")
+    .find("tbody tr")
+    .each(function () {
+      var obj = {},
+        $td = $(this).find("td");
+      currentInput = $td.eq(1).find("input").val();
+      if ((currentInput == null || currentInput == "") && isExcel == false) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=5394149)